packages/client/src/cards.tsx

```diff
@@ -0,0 +1,23 @@
+import { range } from 'lodash';
```

Comment:
Don't know about conventions here, wondering if having a file named `Card` and one called `cards` gets confusing. Maybe `deck`? Don't know, don't care that much.

Reply:
Yeah we will think about a better structure later, this is temporary anyway